src/test/fuzz/tx_pool.cpp

```diff
@@ -29,6 +29,15 @@ struct MockedTxPool : public CTxMemPool {
     }
 };
 
+class DummyChainState final : public CChainState
+{
+public:
```

Comment:
I'd prefer to declare this as a class, even if it requires an additional line for the access specifier (since it's a class and not just a data structure).